packaging/windows.cmake

```diff
@@ -60,7 +60,8 @@ set(CPACK_RESOURCE_FILE_LICENSE ${CMAKE_SOURCE_DIR}/LICENSE)
 set(CPACK_PACKAGE_CONTACT "Arjen Hiemstra <a.hiemstra@ultimaker.com>")
 
 set(CPACK_PACKAGE_EXECUTABLES Cura "Cura ${CURA_VERSION_MAJOR}.${CURA_VERSION_MINOR}.${CURA_VERSION_PATCH}")
-set(CPACK_PACKAGE_INSTALL_DIRECTORY "Cura ${CURA_VERSION_MAJOR}.${CURA_VERSION_MINOR}")
+set(CPACK_PACKAGE_INSTALL_DIRECTORY "Cura")
+set(CPACK_PACKAGE_INSTALL_REGISTRY_KEY "Cura")
```

Comment:
This prevents users from installing multiple versions alongside each other. I know it's not common, but historically this is what our user base wants to be able to do, so I don't think we should change that right now.

Reply:
Installing multiple versions is wanted? I better have one version installed, which works, instead of multiple ones, which are all working good in their own way..